marklogic-data-hub-central/ui/src/components/zero-state-explorer/zero-state-explorer.test.tsx

```diff
@@ -50,4 +50,22 @@ describe("zero state explorer component", () => {
     fireEvent.click(exploreButton);
     expect(exploreButton.onclick).toHaveBeenCalledTimes(1);
   });
+
+  test("Verify entity dropdown option auto selection", () => {
+    const {getByText, queryByText} = render(<ZeroStateExplorer entities={entities} setEntity={jest.fn()} isSavedQueryUser={true} queries={queries} hasStructured={false} setCardView={jest.fn()} columns={columns} setIsLoading={jest.fn()} tableView={true} toggleTableView={jest.fn()} setDatabasePreferences={jest.fn()}/>);
+    const finalButton = document.querySelector("[aria-label=\"switch-database-final\"]");
```

Comment:
Test works fine. However, I'd recommend using the native RTL query methods(**getByLabel** in this case) wherever possible instead of document.querySelector as a best practice. Those methods have better bindings and support with react testing library.

Reply:
Fixed and rebased. Thanks.